cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -519,3 +519,34 @@ def test_duplicate_mapping_name(appliance, mapping_data_vm_obj_mini):
     view.general.description.fill("description")
     view.general.flash.assert_message(f"Infrastructure mapping {name} already exists")
     view.general.cancel_btn.click()
+
+
+def test_migration_with_no_conversion(appliance, provider, mapping_data_vm_obj_mini):
+    """
+    Test Migration plan without setting conversion hosts
+
+    Polarion:
+        assignee: sshveta
+        initialEstimate: 1/2h
+        caseimportance: high
+        caseposneg: negative
+        testtype: functional
+        startsin: 5.10
+        casecomponent: V2V
+    """
+    # Delete existing conversion entries from cloudforms
+    delete_hosts = appliance.ssh_client.run_rails_command(
```

Comment:
I would expect this to be in a fixture. Since it is cleaning up the appliance in order to run the test. Could be a global fixture like `has_no_providers`, something like `has_no_conversion_hosts`. What do you think?